src/main/resources/templates/project.html

```diff
@@ -35,6 +35,10 @@ <h3>Students:</h3>
     <div>
         <hr/>
         <h3>Oral Presentation</h3>
+        <div th:style="${selectedTime != null} ? 'display:block' : 'display:none'">
```

Comment:
use `th:unless="${selectedTime == null}"`

Reply:
that doesn't work because i need to actively assign the style

Reply:
I'd need an if else statement

Reply:
i was wrong
i fixed it